core/domain/takeout_service_test.py

```diff
@@ -795,7 +828,7 @@ def test_export_data_for_full_user_trivial_is_correct(self):
         subscriptions_data = {
             'activity_ids': [],
             'collection_ids': [],
-            'creator_usernames': [],
+            'feedback_thread_ids': [],
```

Comment:
Why did this change?

Reply:
to me this seems like data pertaining to other users, so it should not be exported. (also as a side note: it does not fit nicely with the schema that we have). Please let me know if I am mistaken!

Reply:
Discussed offline -- @varun-tandon will make a small change to incorporate this since we do want to export the usernames of the folks which the user subscribed to.

Reply:
Done